conference/management/commands/create_initial_data_for_dev.py

```diff
@@ -4,14 +4,14 @@
 
 from django.core.management.base import BaseCommand
 from django.conf import settings
+from django.db import transaction
 from django.utils import timezone
 from django.core.exceptions import FieldError
 
 from cms.api import create_page
-from conference.models import Conference
 from assopy.models import Vat
 from conference.fares import pre_create_typical_fares_for_conference
-
+from tests.common_tools import create_homepage_in_cms
```

Comment:
Hmm, I don't think it's a good idea to have 'real' code depend on tests :neutral_face:

Reply:
I agree - these are development fixtures though, so not sure they should be considered `real` code. I'm happy to move them to a `fixture` module if you think that makes things clearer.

Reply:
I think in this case it can work, because it's about provisioning local dev data for  – wait for it – testing purposes. :)